lib/CL/pocl_file_util.h

```diff
@@ -24,11 +24,30 @@
 #ifndef POCL_FILE_UTIL_H
 #define POCL_FILE_UTIL_H
 
+#include "pocl_export.h"
 
 #ifdef __cplusplus
+#include <cstdint>
 extern "C" {
+#else
+#include <stdint.h>
 #endif
 
+  typedef enum pocl_file_type_e
```

Comment:
Is the indentation wrong here due to the extern "C" braces?

Reply:
Could be my mistake. I don't think I have run the formatter over headers.

Reply:
According to clang-format-17, --style=GNU wants to indent the line (perhaps due to the extern "C"). I'll fix it manually.

Reply:
Addressed.